vdev/src/testing/integration.rs

```diff
@@ -28,11 +29,28 @@ impl IntegrationTest {
         let integration = integration.into();
         let environment = environment.into();
         let (test_dir, config) = IntegrationTestConfig::load(&integration)?;
+        // Load thee compose_config
+        let mut compose_config = ComposeConfig::parse(&test_dir.join("compose.yaml"))?;
+        let network_name = format!("vector-integration-tests-{integration}");
+        // Inject the networks block
+        // TODO: Inject the labels block
+        compose_config.networks.insert("default".to_string(), {
+            let mut default_network = BTreeMap::new();
+            default_network.insert("name".to_string(), network_name.clone());
+            default_network
```

Comment:
There is a `from_iterator` that can simplify this:
```suggestion
            BTreeMap::from_iterator([(name".to_string(), network_name.clone())])
```